packages/summer-earn-protocol-subgraph/src/index.ts

```diff
@@ -0,0 +1,54 @@
+import { UserPositionsQuery, getSdk } from './client';
+import { GraphQLClient } from 'graphql-request';
+import { ChainId } from '@summerfi/serverless-shared'
+
+interface SubgraphClientConfig {
+    chainId: ChainId
+    urlBase: string
+}
+
+export interface GetUserPositionsParams {
+    userAddress: string
+}
+
+const chainIdSubgraphMap: Partial<Record<ChainId, string>> = {
+    [ChainId.BASE]: 'summer-protocol-base',
+}
+
+const supportedChains = Object.keys(chainIdSubgraphMap).map((k) => parseInt(k))
+
+const getClient = (chainId: ChainId, baseUrl: string) => {
+    if (!supportedChains.includes(chainId)) {
+        throw new Error(`ChainId ${chainId} not supported
+        Supported chains: ${supportedChains}`)
+    }
+    const subgraph = chainIdSubgraphMap[chainId]
+    if (!subgraph) {
+        throw new Error(`No subgraph for chainId ${chainId}`)
+    }
+    const url = `${baseUrl}/${subgraph}`
+    const client = new GraphQLClient(url);
+    return getSdk(client);
```

Comment:
Is this SDK related somehow to our SDK?

Reply:
no. it's a method in the autogenerated graphql cleint :)